mins/src/run_subscribe.cpp

```diff
@@ -80,11 +80,6 @@ int main(int argc, char **argv) {
   op->sys->save_timing ? save->save_timing_to_file(sys->tc_sensors->get_total_sum()) : void();
   save->check_files();
 
-  // Call destructor for a cleaner termination
-  sys->~SystemManager();
-  pub->~ROSPublisher();
-  op->~Options();
-  save->~State_Logger();
```

Comment:
Hi! Thanks for looking at this problem!
I also actually looked into this and ended up having these explicit destructors.
Previously, when the system was terminated successfully, I got the following error message:
```
terminate called after throwing an instance of 'class_loader::LibraryUnloadException'
  what():  Attempt to unload library that class_loader is unaware of.
```
This error also shows up in OpenVINS, and I tried to resolve this issue ended up having those explicit destructors - SystemManager destructor seems to remove the error message.
I was aware of the double destruction issue, but at least I had a clean system termination on my terminal so left it as it was.
Will this double destruction be an issue?

Reply:
I see. I also find that removing explicit destructors will get these messages:
![image](https://github.com/rpng/MINS/assets/23168485/d8b47ebd-eb8c-4021-a70c-32ba2583e057)

I think this error might be from image_transport. And the error really doesn't show on screen by calling destructors explicitly. But I think the error doesn't disappear, it might be hidden by error "corrupted double-linked list", which I think is more severe than 'class_loader::LibraryUnloadException'.

Reply:
Okay, thank you for sharing your opinion. If you are finished with improving the compatibility of MINS, I will go through the test and merge your PR if all is good.

Reply:
Yes, I have done some tests and it seems that everything runs correctly and natively on my PC. And CI/CD also passes: [action](https://github.com/lnexenl/MINS/actions/runs/6469208222)